ComputeRawCF.py

```diff
@@ -55,58 +55,85 @@
 
 hSE = {}
 hME = {}
+hMErew = {}
 for comb, fdcomb in combs.items():
     oFile.mkdir(comb)
     oFile.cd(comb)
-
+    
     hSE[comb] = {}
     hME[comb] = {}
+    hMErew[comb] = {}
     for region in regions:
-        hSE[comb][region] = Load(inFile, f'HMResults0/HMResults0/{fdcomb}/SEDist_{fdcomb}')
-        hME[comb][region] = Load(inFile, f'HMResults0/HMResults0/{fdcomb}/MEDist_{fdcomb}')
-
+        pcsuffix = cfg['pcsuffix']
```

Comment:
In general the suffix can be also related to other things, not just the pair cleaner
```suggestion
        runSuffix = cfg['runsuffix']
```